### PR TITLE
Adding preload html attribute to video component

### PR DESCRIPTION
**Pull request by anaghav2023**
## Description
Refer to this Jira Ticket to understand the issue: https://builder-io.atlassian.net/browse/SUPP-24

Or this loom: https://www.loom.com/share/1d3b0696a6d84c3cac2d7273641d7e20

**Loom**
Watch this loom to see the sdk changes working
https://www.loom.com/share/27f904ae6c18410bbe3c90aaaed523bb

### CODE REVIEW

File: packages/sdks/src/blocks/video/video.lite.tsx [hunk 1]
```diff
@@ -47,6 +48,7 @@ export default function Video(props: VideoProps) {
   return (
     <video
       {...state.spreadProps}
+      preload={props.preload}
```
Comment (teleaziz) [inline packages/sdks/src/blocks/video/video.lite.tsx:51]:
```suggestion
      preload={props.preload || 'metadata'}
```

Default values on inputs do not apply retroactively, meaning that content that is already published will have this as undefined

File: packages/sdks/src/blocks/video/component-info.ts [hunk 2]
```diff
@@ -57,6 +57,12 @@ export const componentInfo: ComponentInfo = {
       defaultValue: 'cover',
       enum: ['contain', 'cover', 'fill', 'auto'],
     },
+    {
+      name: 'preload',
+      type: 'text',
+      defaultValue: 'auto',
```
Comment (teleaziz) [inline packages/sdks/src/blocks/video/component-info.ts:63]:
```suggestion
      defaultValue: 'metadata',
```

Wdyt if we match the spec, default value in browsers is `metadata` https://developer.mozilla.org/en-US/docs/Learn/Performance/video#video_preload

**Review by teleaziz — CHANGES_REQUESTED** [review]

File: packages/sdks/src/blocks/video/video.lite.tsx (hunk 1 above)
Comment (anaghav2023) [reply packages/sdks/src/blocks/video/video.lite.tsx:51]: Oh right ! Thanks. So Ill also add it to the Video.tsx file in react sdk

File: packages/sdks/src/blocks/video/component-info.ts (hunk 2 above)
Comment (anaghav2023) [reply packages/sdks/src/blocks/video/component-info.ts:63]: okay cool

**Review by teleaziz — APPROVED** [review]

**Pull request merged**